### PR TITLE
moves repeated blog section into template

### PR DESCRIPTION
**Pull request by mdrillin**
Addresses issue #551 

- add 'About Debezium' and 'Get involved' sections into the blog post template, underneath the author block
- remove the repeated section from all of the blog post asciidocs

### CODE REVIEW

File: _layouts/post.html [hunk 1]
```diff
@@ -122,6 +122,32 @@ <h2>{{ authorName }}</h2>
       
     
     {% endif %}
+    <div class="row">
```
Comment (indraraj) [inline _layouts/post.html:125]: This need to be moved above the "about author card"

**Comment by indraraj** [comment]
Tested locally, looks good. Just need to place it above the author card to match the current site.

**Comment by gunnarmorling** [comment]
In fact I had suggested to put it below the author card as right now there's no clear boundary between the actual post and that standard "footer". How does it look, perhaps you could attach a screenshot?

**Comment by indraraj** [comment]
image

 @gunnarmorling

**Comment by indraraj** [comment]
sry, I missed reading the complete description of the Github issue. I see that it is something you actually asked for, you can go ahead and merge this @mdrillin

**Review by indraraj — APPROVED** [review]

**Comment by gunnarmorling** [comment]
Thanks, so what do you which one is better? Perhaps the two footer paragraphs should be styled slightly different than the main text?

**Comment by indraraj** [comment]
I think it's good to move it below the "About author" card as it's not part of the blog content. Yes, different styling will be great but for now, we can put some blog end marker/separator. wdyt?

image

**Comment by gunnarmorling** [comment]
Yes, good idea. 👍

**Comment by mdrillin** [comment]
Updated the commit to include separator - merging.

**Pull request merged**